docs/analytics.md

```diff
@@ -161,9 +161,16 @@ statistics, etc.). There is currently a reference implementation of a
 ### Setup and Testing
 
 The main testing approach for the /stats page UI is manual testing.  For UI
+<<<<<<< HEAD
 testing, you want a comprehensive initial data set; you can use `manage.py
 populate_analytics_db` to set up, login as the shylock user, and then go to
 /stats.
+=======
+testing, you want a comprehensive initial data set;
+you can run the commands (Make sure you're in the zulip directory):
+- `./manage.py populate_analytics_db`,
+- `./tools/run-dev.py` to set up, then, while logging in choose "shylock@analytics.ds", and then go to /stats.
```

Comment:
Please wrap this line properly. Also it will be good to change `set up` -> `start the server`. :)